models/ticket.go

```diff
@@ -12,10 +20,80 @@ type Ticket struct {
 	ScreenShot []ScreenShot `json:"screenshots"`
 }
 
+type HoneyclientRequest struct {
```

Comment:
This doesn’t need to be its own type, we can just pass the ticket object as is

Reply:
just updated this